sunpy/coordinates/utils.py

```diff
@@ -469,3 +470,62 @@ def get_limb_coordinates(observer, rsun: u.m = constants.radius, resolution=1000
                     frame='heliocentric',
                     observer=observer, obstime=observer.obstime)
     return limb
+
+
+def get_heliocentric_angle(coordinate_on_solar_disk):
+    r"""
+    The heliocentric angle is the angle between direction from a point on the surface of the body toward the observer, with respect to the local vertical of that body.
+
+    The range of value is between 0 :math:`^\circ` (at the center) and 90 :math:`^\circ` (at the edge).
+    If one takes the :math:`\cos` of this value, this is referred to as :math:`\mu`.
+
+    It is worked out using the following equation:
+
+    .. math::
+
+        heliocentric angle = \arcsin(\frac{d}{r})
+        \mu=\cos(heliocentric angle)
+
+    where :math:`d` is the distance between the observer and the point on the solar surface and :math:`r` is the radius of the disk.
+
+    Parameters
+    ----------
+    coordinate_on_solar_disk : `astropy.coordinates.SkyCoord`
+        A coordinate on the solar disk, requires the observer and obstime to be set.
+
+    Returns
+    -------
+    heliocentric_angle : `~astropy.units.Quantity`
+        The angle between the local solar vertical and the line of sight from
+        the observer to a point on the solar disk.
+
+    Examples
+    --------
+    >>> import numpy as np
+
+    >>> import astropy.units as u
+    >>> from astropy.coordinates import SkyCoord
+
+    >>> from sunpy.coordinates.utils import get_heliocentric_angle
+
+    >>> # At the center of the solar disk
+    >>> hpc_coord_center = SkyCoord(0*u.arcsec, 0*u.arcsec, frame='helioprojective', observer="earth", obstime="2017-07-26")
+    >>> get_heliocentric_angle(hpc_coord_center)
+    <Quantity 0. deg>
+    >>> # Mu
+    >>> np.cos(get_heliocentric_angle(hpc_coord_center).to(u.radian)).to_value()
```

Comment:
This feels more complex than it needs to be, you don't need to convert to radian, and why have the `to_value`?

```suggestion
    >>> np.cos(get_heliocentric_angle(hpc_coord_center))
```

---

This works ftr:

```
>>> np.cos(10*u.deg)
<Quantity 0.98480775>
>>> np.cos((10*u.deg).to(u.radian))
<Quantity 0.98480775>
```